production/helm/values.yaml

```diff
@@ -21,11 +23,14 @@ loki:
     labels: {}
 
   readinessProbe: {}
+    # httpGet:
```

Comment:
I think the readinessProbe and livenessProbe should both be enabled by default.

Reply:
was there a reason not to add a livenessProbe?